### PR TITLE
systemtest.CleanupElasticsearch: refresh indices

### PR DESCRIPTION
**Pull request by axw**
## Motivation/summary

Prior to deleting the ILM policy, refresh indices to ensure the most recent changes are visible.

See https://github.com/elastic/apm-server/issues/4366#issuecomment-728676137. A little bit of a shot in the dark.

## Checklist

- [x] I have signed the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/).
~- [ ] I have updated [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/master/CHANGELOG.asciidoc)~

I have considered changes for:
~- [ ] documentation~
~- [ ] logging (add log lines, choose appropriate log selector, etc.)~
~- [ ] metrics and monitoring (create issue for Kibana team to add metrics to visualizations, e.g. [Kibana#44001](https://github.com/elastic/kibana/issues/44001))~
- [x] automated tests (add tests for the code changes, all [**unit** tests](https://github.com/elastic/apm-server/blob/master/TESTING.md) pass locally)
~- [ ] telemetry~
~- [ ] Elasticsearch Service (https://cloud.elastic.co)~
~- [ ] Elastic Cloud Enterprise (https://www.elastic.co/products/ece)~
~- [ ] Elastic Cloud on Kubernetes (https://www.elastic.co/elastic-cloud-kubernetes)~

## How to test these changes

`cd systemtest && go test -v`

## Related issues

https://github.com/elastic/apm-server/issues/4366

### CODE REVIEW

**Review by jalvz — APPROVED** [review]
shoot!

File: systemtest/elasticsearch.go [hunk 1]
```diff
@@ -95,6 +95,7 @@ func CleanupElasticsearch(t testing.TB) {
 		esapi.IndicesDeleteRequest{Index: []string{prefix}},
 		esapi.IngestDeletePipelineRequest{PipelineID: prefix},
 		esapi.IndicesDeleteTemplateRequest{Name: prefix},
+		esapi.IndicesRefreshRequest{},
```
Comment (simitt) [inline systemtest/elasticsearch.go:98]: This should be called _after_ the other requests are done and _before_ the ILM request is done. In https://github.com/elastic/apm-server/pull/4434/files#diff-1b744e35d185e772edc9c58ab2f27cd51c987e60bbc821d71781b06769d7df34R112 however the requests are run in parallel, so I believe this should be after these requests.

Comment (axw) [reply systemtest/elasticsearch.go:98]: Indeed, thanks :man_facepalming:

**Review by simitt — APPROVED** [review]

**Comment by axw** [comment]
Failure due to an unrelated flaky test, opened another issue.

**Pull request merged**